bundles/org.openhab.binding.ventaair/src/main/java/org/openhab/binding/ventaair/internal/VentaAirDeviceConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ventaair.internal;
+
+import java.math.BigDecimal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link VentaAirDeviceConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Stefan Triller - Initial contribution
+ */
+@NonNullByDefault
+public class VentaAirDeviceConfiguration {
+    public @Nullable String ipAddress;
+    public @Nullable String macAddress;
+    public @Nullable BigDecimal deviceType;
+    public @Nullable BigDecimal hash;
+    public @Nullable BigDecimal pollingTime;
```

Comment:
```suggestion
    public String ipAddress="";
    public String macAddress="";
    public BigDecimal deviceType =BigDecimal.ZERO;
    public BigDecimal hash =BigDecimal.ZERO;
    public BigDecimal pollingTime=BigDecimal.ZERO;
```
Doing this means you don't need to check for null anymore.